documentation/manual/src/main/paradox/algebras-and-interpreters.md

```diff
@@ -105,24 +105,25 @@ work with JSON.
-| @ref[Akka HTTP](interpreters/akka-http.md)                | Client and server backed by [Akka HTTP](https://doc.akka.io/docs/akka-http/current/)                          |
-| @ref[http4s](interpreters/http4s.md)                      | Client and server backed by [http4s](https://http4s.org)                                                      |
-| [Play framework](https://endpoints.github.io/play)        | Client and server backed by [Play framework](https://www.playframework.com/)                                  |
-| @ref[Scala.js web (XHR)](interpreters/scalajs-web-xhr.md) | Scala.js web client using `XMLHttpRequest`                                                                    |
-| @ref[Scala.js web (Fetch)](interpreters/scalajs-web-fetch.md) | Scala.js web client using `Fetch`                                                                             |
-| [scalaj-http](https://endpoints4s.github.io/scalaj)       | JVM client backed by [scalaj-http](https://github.com/scalaj/scalaj-http)                                     |
-| @ref[sttp](interpreters/sttp.md)                          | JVM client backed by [sttp](https://github.com/softwaremill/sttp)                                             |
-| @ref[OpenAPI](interpreters/openapi.md)                    | Generates [OpenAPI](https://github.com/OAI/OpenAPI-Specification) documents for endpoints definitions         |
-| @ref[circe](interpreters/circe.md)                        | Builds [circe](http://circe.github.io/circe/) codecs out of JSON schema definitions                           |
-| @ref[Play JSON](interpreters/play-json.md)                | Builds [Play JSON](https://github.com/playframework/play-json) Reads and Writes out of JSON schema definitions |
+| Family                                                        | Description                                                                                                    |
+|---------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|
+| @ref[Pekko HTTP](interpreters/pekko-http.md)                  | Client and server backed by [Pekko HTTP](https://pekko.apache.org/docs/pekko/current/)                         |
+| [Akka HTTP](https://endpoints4s.github.io/akka-http)          | Client and server backed by [Akka HTTP](https://doc.akka.io/docs/akka-http/current/)                           |
+| @ref[http4s](interpreters/http4s.md)                          | Client and server backed by [http4s](https://http4s.org)                                                       |
+| [Play framework](https://endpoints4s.github.io/play)          | Client and server backed by [Play framework](https://www.playframework.com/)                                   |
```

Comment:
Thank you for fixing this link along the way! :pray:

This is great, thank you SO MUCH for all the work! I think the last thing we should update in the docs should be [the index page](https://github.com/endpoints4s/endpoints4s/blob/master/documentation/manual/src/main/paradox/index.md) to include Pekko HTTP.

Reply:
All good, that reminded me to add Akka back to the list 😄 I mostly find-and-replaced entries in docs, so we found some mistakes.